doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_flpr.rst

```diff
@@ -0,0 +1,175 @@
+   The FLPR core support in the |NCS| is currently :ref:`experimental<software_maturity>`.
+
+The nRF54H20 SoC includes a dedicated VPR CPU, based on RISC-V architecture, known as the *fast lightweight peripheral processor* (FLPR).
+The FLPR core can be used to manage specific peripherals through the appropriate Zephyr Device Driver API:
+
+* ##List to be completed
+
+.. _vpr_flpr_nrf54h20_initiating:
+
+Using FLPR with Zephyr multithreaded mode
+*****************************************
+
+The FLPR core can function as a general-purpose core, running under the full Zephyr kernel.
+Building the FLPR target is similar to building the application core, but the application core build must include an overlay that enables the FLPR core.
+
+Bootstrapping the FLPR core
+===========================
+
+The |NCS| provides a FLPR snippet that adds the overlay needed for bootstrapping the FLPR core.
+The primary purpose of this snippet is to enable the transfer of the FLPR code to the designated memory region (if required) and initiate the FLPR core.
+
+When building for the ``nrf54h20dk/nrf54h20/cpuflpr`` target, a minimal sample is automatically loaded onto the application core.
+For more details, see :ref:`building_nrf54h_app_flpr_core`.
+
+Memory allocation
+*****************
+
+Running the FLPR CPU can lead to increased latency when accessing ``RAM_21``.
+To mitigate this, you should use ``RAM_21`` exclusively for FLPR code, FLPR data, and non-time-sensitive information from the application CPU.
+For data that requires strict access times, such as DMA buffers and application CPU data used in low-latency ISRs, you should use ``RAM_0x``.
+
+.. _building_nrf54h:
+
+Building and programming with the nRF54H20 DK
+*********************************************
+
+.. note::
+   The FLPR core support in the |NCS| is currently :ref:`experimental<software_maturity>`.
+
+Depending on the sample, you may need to program only the application core or both the FLPR and application cores.
+Additionally, the process will vary depending on whether you are working with a single-image or multi-image build.
+
+.. note::
+   The following instructions do not cover the scenario of multi-image single-core builds.
+
+Building for the application core only
+======================================
+
+Building for the application core follows the default building process for the |NCS|.
+For detailed instructions, refer to the :ref:`building` page.
+
+.. _building_nrf54h_app_flpr_core:
+
+Building for both the application and FLPR core
+===============================================
+
+Building for both the application core and the FLPR core differs from the default |NCS| procedure.
+Additional configuration is required to enable the FLPR core.
+
+This section explains how to build and program both cores, covering separate builds and sysbuild configurations.
+The FLPR core supports two variants:
+
+* ``nrf54h20dk/nrf54h20/cpuflpr``: FLPR runs from SRAM (recommended method).
```

Comment:
Being precise here would help to understand the partitioning recommendations described earlier.

```suggestion
* ``nrf54h20dk/nrf54h20/cpuflpr``: FLPR runs from RAM_21 (recommended method).
```